examples/gke-a4-highgpu/gke-a4-highgpu-deployment.yaml

```diff
@@ -12,20 +12,34 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
----
 terraform_backend_defaults:
   type: gcs
   configuration:
-    bucket: BUCKET_NAME
+    # The GCS bucket used for storing terraform state
+    bucket:
```

Comment:
nit: Add an inline comment to ask customer to add this. Applicable everywhere else in the blueprint
Like this:
`bucket: #add this`

Reply:
The GKE GCS blueprint recommends this format which is followed for A4 High as well: https://github.com/GoogleCloudPlatform/cluster-toolkit/blob/main/examples/hypercompute_clusters/a3u-gke-gcs/deployment.yaml